View Controllers/TimelineViewController.m

```diff
@@ -82,7 +82,6 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     NSURL *url = [NSURL URLWithString:URLString];
     NSData *urlData = [NSData dataWithContentsOfURL:url];
     cell.profilePictureView.image = [UIImage imageWithData:urlData];
-    
```

Comment:
I might even leave this tbh. It's nice sometimes to have a line of separation between doing stuff to something and returning that thing.